### PR TITLE
TODOs: 12, 13, 14

### PR DESCRIPTION
**Pull request by pwasowski2**
Signed-off-by: Pawel Wasowski <p.wasowski2@samsung.com>

### Change Description ###

Add guides for:
- registerService()
- unregister()
- unregisterAllServices()
- notifyAboutValueChange()

### CODE REVIEW

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 1]
```diff
@@ -755,24 +755,208 @@ To read and write a value of the Bluetooth descriptor:
...
+
+   var characteristicInitData = {
+     uuid: '9876',
+     descriptors: [descriptorInitData]
+     /* Define more attributes, if needed */
+   };
+
+   var includedServiceInitData = {
+     serviceUuid: '5678',
+     isPrimary: true
```
Comment (djuszczakk) [inline docs/application/web/guides/connectivity/bluetooth.md:780]: is it possible to create primary service which is included in other primary service?

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 2]
```diff
@@ -755,24 +755,208 @@ To read and write a value of the Bluetooth descriptor:
...
+   var server = tizen.bluetooth.getGATTServer();
+   server.registerService(serviceInitData, successCallback, errorCallback);
+   ```
+   > **Note**
+   > The service will be registered in the server until one of the following
+   > happens:
+   > - service is unregistered with its `unregister()` method,
+   > - all services are unregistered with server's `unregisterAllServices()` method,
+   > - application is reloaded,
+   > - application is closed.
```
Comment (djuszczakk) [inline docs/application/web/guides/connectivity/bluetooth.md:810]: I think that if app is closed you can't execute any code from that app.

```suggestion
   > - application was closed.
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 3]
```diff
@@ -755,24 +755,208 @@ To read and write a value of the Bluetooth descriptor:
...
+
+   serviceToBeUnregistered.unregister(successCallback, errorCallback);
+   ```
+   > **Note**
+   > After `unregister()` is called for the last registered service, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+To unregister all services from the local GATT server at once, call server's
+`unregisterAllServicesMethod()`:
```
Comment (djuszczakk) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
`unregisterAllServices()` method:
```

**Review by djuszczakk — CHANGES_REQUESTED** [review]
Please apply your changes to another branch (tizen_6.0_prepare) on this repository

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 1 above)
Comment (pwasowski2) [reply docs/application/web/guides/connectivity/bluetooth.md:780]: Yes, it's possible. As far as I remember, the only significant difference between primary and secondary services is that secondary services cannot include primary ones. Primary services can include both primary and secondary services.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 2 above)
Comment (pwasowski2) [reply docs/application/web/guides/connectivity/bluetooth.md:810]: 1. Yes, the user can't execute any code. The service will be unregistered automatically, when app is closed - i.e. clients won't be able to access it.

2. Why should we change "is" to "was"? Currently it's a sentence in the first conditional https://learnenglish.britishcouncil.org/grammar/intermediate-to-upper-intermediate/conditionals-1

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 4]
```diff
@@ -755,24 +755,207 @@ To read and write a value of the Bluetooth descriptor:
 ### Accessing connection's ATT MTU
 <!-- END #11 -->
 
-<!-- TODO #12
-      - write about registerService()
- -->
```
Comment (arkadiuszpietraszek) [inline docs/application/web/guides/connectivity/bluetooth.md:760]: We leave this part in other PR for convenience of merging them all at the end.

**Review by arkadiuszpietraszek — APPROVED** [review]

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 4 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:760]: Agree wiith Arek. We will remove commented TODO sections in the last commit, after all other PRs will be merged.
Please do not remove any of them in your changes,

**Review by djuszczakk — CHANGES_REQUESTED** [review]

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 4 above)
Comment (pwasowski2) [reply docs/application/web/guides/connectivity/bluetooth.md:760]: Done

**Review by djuszczakk — APPROVED** [review]

**Pull request merged**